### PR TITLE
Swathi/meetingid1.1

### PR DESCRIPTION
**Pull request by swathishanmugam**
Removed meetingid field and added meetingname.
updated auto-generated meeting id on the admin_main page.

### CODE REVIEW

**Review by AkalyaAsokan — APPROVED** [review]

File: src/components/admin/create-meeting.js [hunk 1]
```diff
@@ -73,7 +74,8 @@ function CreateMeeting() {
 
     function handleTextInputChange(event) {
         const value = event.target.value
-        setMeetingId(value)
+        // setMeetingId(value)
```
Comment (linda-huang) [inline src/components/admin/create-meeting.js:77]: Hey, can we please remove these commented out lines? For the rest of the files as well. Thanks!

File: src/components/admin/admin-main.js [hunk 2]
```diff
@@ -363,7 +363,7 @@ function AdminMain() {
     useEffect(() => {
         const interval = setInterval(() => {
             if (MeetingActive) {
-                setMeetingId(location.state.meetingId);
+                // setMeetingId(location.state.meetingId);
```
Comment (linda-huang) [inline src/components/admin/admin-main.js:366]: Here as well.

File: src/components/admin/create-meeting.js [hunk 3]
```diff
@@ -5,7 +5,8 @@ import { restUrl } from "../..";
 function CreateMeeting() {
 
     const navigate = useNavigate();
-    const [meetingId, setMeetingId] = useState("");
+    // const [meetingId, setMeetingId] = useState("");
```
Comment (linda-huang) [inline src/components/admin/create-meeting.js:8]: Ditto.

**Review by linda-huang — COMMENTED** [review]
Looks good! If you can remove the commented-out lines of code, that'd be great.

**Review by AkalyaAsokan — APPROVED** [review]

**Review by linda-huang — APPROVED** [review]

**Pull request merged**